src/main/scala/chisel3/Driver.scala

```diff
@@ -149,7 +149,11 @@ object Driver extends BackendCompilationUtilities {
     /* create custom transforms by finding the set of transform classes associated with annotations
      * then instantiate them into actual transforms
      */
-    val transforms = circuit.annotations.map(_.transform).toSet.map { transformClass: Class[_ <: Transform] =>
+    val transforms = circuit.annotations
+                            .map(_.transform)
+                            .distinct
+                            .filterNot(_ == classOf[firrtl.Transform])
```

Comment:
I'm not sure what that filterNot is doing, removing transforms that firrtl implements perhaps.  If so shouldn't that be done in firrtl, seems like firrtl.Transform might be too specific.